feature/announcement/src/main/res/layout/item_announcement.xml

```diff
@@ -48,6 +48,8 @@
             android:layout_height="wrap_content"
             android:layout_marginTop="14dp"
             android:textColor="?colorOnBackground"
+            android:textColorLink="@color/design_default_color_secondary"
+            android:autoLink="all"
```

Comment:
Can you use autolink=web? because probably we don't use a phone number, mail address 👍

Reply:
I agreed. Phone numbers, mail addresses, and map addresses are needless for now.

By the way, *Linkify.MAP_ADDRESSES* (and also Linkify.ALL of course) may cause a crash on devices that do not have Webkit. Android devices require `WebView#findAddress` to grep Geo URLs and throw a ClassNotFoundException if it's missing.

Reply:
I resolved it !